### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Tantomile**
A nice completed page to be shown to anyone who visits the primary github.com/TheCookieMC/ page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
-# .github
-Readme information for TheCookieMC
+## Welcome to TheCookieMC!
+
+![CookieMC banner](https://raw.githubusercontent.com/TheCookieMC/BugReports/main/cookiemc.png?token=GHSAT0AAAAAABZTNNIHRBVS66ROEBB6OHVSY3O5CUA)
```
Comment (BillyDotWS) [inline README.md]: Not a publicly visible link, please provide a link on a public image hosting website.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+
+Hello and Welcome to the offical github page for TheCookieMC! Here, you will find all of our API's information and documentation!
```
Comment (BillyDotWS) [inline README.md]: "Hello and Welcome", doesn't sound like the best way to introduce people.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
-# .github
-Readme information for TheCookieMC
+## Welcome to TheCookieMC!
+
```
Comment (BillyDotWS) [inline README.md:3]: Shouldn't be a blank line here, not needed.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,27 @@
...
+![CookieMC banner](https://raw.githubusercontent.com/TheCookieMC/BugReports/main/cookiemc.png?token=GHSAT0AAAAAABZTNNIHRBVS66ROEBB6OHVSY3O5CUA)
+
+Hello and Welcome to the offical github page for TheCookieMC! Here, you will find all of our API's information and documentation!
+
+### What is TheCookieMC?
+
```
Comment (BillyDotWS) [inline README.md]: Shouldn't be a blank line here, not needed.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,27 @@
...
+TheCookieMC is a Java minigame network!
+
+We started development in 2019, and after some discussion, we stopped development due to developers not being available and Minecraft declining in players.
+
+In 2021, we saw the quality of other Java servers decline: we had the idea to continue with development, redoing old plugins and making new ones to finish our idea and create a fun server.
+
```
Comment (BillyDotWS) [inline README.md]: This is a github readme, there's no reason for this to be here.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,27 @@
...
+Our server is currently in a closed beta state, we'll sometimes post invites on our discord server to help us test & we may eventually change to a fully fledged server!
+
+### Important Links
+
```
Comment (BillyDotWS) [inline README.md]: Shouldn't be a blank line here, not needed.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,27 @@
...
+IP Address: | play.thecookiemc.net
+<br>Discord Link: | https://discord.gg/TSf6rgx4MG
+<br>Punishment Appeals: | http://thecookiemc.net/appeal
+<br>Twitter: | https://twitter.com/PlayCookieMC
+      
+
+### One last note
+
```
Comment (BillyDotWS) [inline README.md]: Shouldn't be a blank line here, not needed.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,27 @@
...
+Thanks for checking out our page on GitHub! At this time, we do not have any plans to publish any of our work. The only thing that is currently avalible on our github page is information about our [API](https://github.com/TheCookieMC/API). If you have any questions, you can find more information from staff members in our offical [discord server](https://discord.gg/TSf6rgx4MG). Thanks for stopping by :)
```
Comment (BillyDotWS) [inline README.md]: Half of this doesn't need to be specified, an api link and some description that this is where our documentation is, is fine.

**Review by BillyDotWS — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Tantomile) [reply README.md]: okay on it :)

File: README.md (hunk 2 above)
Comment (Tantomile) [reply README.md]: ok on it

File: README.md (hunk 4 above)
Comment (Tantomile) [reply README.md]: Ok

**Comment by Tantomile** [comment]
will update these things in a bit

**Comment by Tantomile** [comment]
Updated to comply with @BillyDotWS's requests.

**Pull request merged**